### ISSUE TITLE
Release 0.5

### ISSUE THREAD

**Issue by jnioche** [DigitalPebble/storm-crawler#110]
List the issues we want to include in the next release and do it

* [Maven] publish src as artefacts #106

@DigitalPebble/committers-crawler what else?

**Comment by jakekdodd** [comment]
The boring stuff, documentation =/

So #88 and #51, and maybe open a new issue for other things to document. Let's rank-order and choose the top 5 items or so? I'd propose

1. Overview
2. FetcherBolt description
3. Parsing/outlink generation
4. Configuration
5. Streams (#51)

**Comment by GuiForget** [comment]
I'll take a stab at the parsing one

**Comment by rkrombho** [comment]
I'll try the Configuration one. May need some help and reviews for certain parameters as I don't yet understand all of them completely. Working on [my fork](https://github.com/rkrombho/storm-crawler/wiki/Configuration) and submit a PR when it's done.
I'll mark unclear parameters with ??? in the beginning of the description. They will require closer reviews by one of you.

**Comment by jnioche** [comment]
Hi @rkrombho. Thanks for volunteering! Let me know when you're done with it and I'll review.

**Comment by rkrombho** [comment]
I'm pretty much done now with documenting Configuration options but it looks like I can't create pull-requests for wiki repositories.
https://github.com/rkrombho/storm-crawler/wiki/Configuration
(formating of the tables is not so nice)

Any proposal on how I should submit this to the project?

**Comment by jnioche** [comment]
Hi

You should be able to edit the WIKI directly. Just add the Configuration
page and create a link from the main page

Thanks

j.

On 10 May 2015 at 12:23, Robert Krombholz <notifications@github.com> wrote:

> I'm pretty much done now with documenting Configuration options but it
> looks like I can't create pull-requests for wiki repositories.
> https://github.com/rkrombho/storm-crawler/wiki/Configuration
> (formating of the tables is not so nice)
>
> Any proposal on how I should submit this to the project?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DigitalPebble/storm-crawler/issues/110#issuecomment-100625587>
> .
>



-- 



* Open Source Solutions for Text Engineering   http://www.digitalpebble.com
<http://www.digitalpebble.com>*
*http://digitalpebble.blogspot.com <http://digitalpebble.blogspot.com/>*
https://twitter.com/digitalpebble

**Comment by jnioche** [comment]
Here's what's left to do before we can release [0.5](https://github.com/DigitalPebble/storm-crawler/milestones/0.5)

* WIKI Overview (#131)  @jnioche
* WIKI FetcherBolt description (#132)
* WIKI Parsing/outlink generation (#124) @GuiForget 
* Documenting Metadata registration (#88) @jakekdodd 
* Change XPath to take a list of regular expressions for a given key (#123) @GuiForget 
* Resource files overridden by ones provided by default in SC (#61) @jnioche

@DigitalPebble/committers-crawler anyone want to take #132?

The last two items (#123 and #61) could be moved to a future release, what do you think?

**Comment by jakekdodd** [comment]
#88 is done

**Comment by jnioche** [comment]
Ok, so waiting for  @GuiForget to comment on #124 and should be OK to release after that

**Comment by jnioche** [comment]
no news from @GuiForget. Release 0.5 done

**Comment by GuiForget** [comment]
Sorry @jnioche I haven't had the time to look at this these past weeks.

**Comment by jnioche** [comment]
no probs @GuiForget !It's just documentation it can be fixed anytime
